ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/remote/RemoteConnectorExecutor.java

```diff
@@ -150,5 +148,4 @@ && getUserRateLimiterMap().get(user.getName()) != null
     }
 
     void invokeRemoteModel(MLInput mlInput, Map<String, String> parameters, String payload, List<ModelTensors> tensorOutputs);
-
```

Comment:
Is this required by checkstyle to remove this line ?  Why `ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/remote/ConnectorUtils.java` requires a new line at line 255?

Reply:
Reverted.